### PR TITLE
Added small-caps symbol characters.

### PR DESCRIPTION
**Pull request by ThatIsAUsername**

### CODE REVIEW

File: src/UI/Art/SpriteArtist/SpriteLibrary.java [hunk 1]
```diff
@@ -546,6 +547,23 @@ public static Sprite getNumbersSmallCaps()
     return numberSpritesSmallCaps;
   }
 
+  /**
+   * This function returns the sprite sheet for symbol characters that go along with
+   * the letter sprites from getLettersSmallCaps().
+   * Returns a Sprite containing the digits 0-9, one number per frame. The image
```
Comment (ThislsAUsername) [inline src/UI/Art/SpriteArtist/SpriteLibrary.java]: Oh, good, it's not just me who does that.

File: src/UI/Art/SpriteArtist/SpriteLibrary.java [hunk 2]
```diff
@@ -612,6 +630,15 @@ else if( Character.isDigit(text.charAt(i)) )
         int letterIndex = text.charAt(i) - '0';
         g.drawImage(getNumbersSmallCaps().getFrame(letterIndex), x, y, menuTextWidth, menuTextHeight, null);
       }
+      else // Assume symbolic
+      {
+        final String charKey = "%./-~,;:!?";
```
Comment (ThislsAUsername) [inline src/UI/Art/SpriteArtist/SpriteLibrary.java]: Parenthesis or at least some sort of bracket would be cool, too

File: src/UI/Art/SpriteArtist/SpriteLibrary.java (hunk 1 above)
Comment (ThatIsAUsername) [reply src/UI/Art/SpriteArtist/SpriteLibrary.java]: Ah, touché.

File: src/UI/Art/SpriteArtist/SpriteLibrary.java (hunk 2 above)
Comment (ThatIsAUsername) [reply src/UI/Art/SpriteArtist/SpriteLibrary.java]: Done

**Pull request merged**